test/browser/components.test.js

```diff
@@ -1771,63 +1771,6 @@ describe('Components', () => {
 		});
 	});
 
-	it('should set component._internal._dom when sCU returns false', () => {
```

Comment:
+1 for removing this test. I think it was one of the last remaining ones where we tested implementation details directly 👍